pkg/cmd/samples/create.go

```diff
@@ -35,10 +35,9 @@ func NewCreateCmd(config *config.Config) *CreateCmd {
 		forceRefresh: false,
 	}
 	createCmd.Cmd = &cobra.Command{
-		Use:       "create <sample> [destination]",
-		ValidArgs: samples.Names(),
```

Comment:
Just occurred to me: one downside of this approach is that the names will no longer autocomplete. Not a blocker but calling it out.

Reply:
Is this autocomplete via tab key? That doesn't seem to work today either.

Reply:
Yea, I think the homebrew installation stuff is busted :(. Probably not a big deal though, this doesn't seem like something people are going to tab complete.